From ca6f975e837cf985f89d8f962fdac036acd240e4 Mon Sep 17 00:00:00 2001
From: Sanjeev Premi <premi@ti.com>
Date: Sun, 15 Mar 2009 22:33:49 +0530
Subject: [PATCH 362/364] PM: cpuidle - fix compile warnings

Fix compiler warnings introduced in the
commit f3bb4e0...
(cherry picked from commit d8d97e884c1bb94c7ca6a21c3541dc79a3013e4d)
---
 arch/arm/mach-omap2/cpuidle34xx.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-omap2/cpuidle34xx.c b/arch/arm/mach-omap2/cpuidle34xx.c
index 6fb03cf..036d8c0 100644
--- a/arch/arm/mach-omap2/cpuidle34xx.c
+++ b/arch/arm/mach-omap2/cpuidle34xx.c
@@ -70,12 +70,14 @@ static int _cpuidle_allow_idle(struct powerdomain *pwrdm,
 				struct clockdomain *clkdm)
 {
 	omap2_clkdm_allow_idle(clkdm);
+	return 0;
 }
 
 static int _cpuidle_deny_idle(struct powerdomain *pwrdm,
 				struct clockdomain *clkdm)
 {
 	omap2_clkdm_deny_idle(clkdm);
+	return 0;
 }
 
 /**
-- 
1.5.6

